zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -752,14 +752,43 @@ function NotebookCtrl ($scope, $route, $routeParams, $location, $rootScope,
 
   $scope.savePermissions = function () {
     convertPermissionsToArray()
+    if ($scope.isOwnerEmpty()) {
+      BootstrapDialog.show({
+        closable: false,
+        title: 'Setting Owners Permissions',
+        message: 'Please fill the [Owners] field. If not, it will set as current user.\n\n' +
+          'Current user : [ ' + $rootScope.ticket.principal + ']',
```

Comment:
not sure if we need to check $rootScope.ticket.principal  is empty or whitespace